modules/ROOT/pages/security.adoc

```diff
@@ -0,0 +1,11 @@
+= Anypoint Platform Security
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Platform provides security in several layers:
+
+* Manage access for users, groups, environments, client providers, identity providers, connected apps, and external access with xref:access-management::index.adoc[the Access Management user interface].
+You can also manage access programmatically with https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/[Access Management APIs].
```

Comment:
Same comment as previously

Reply:
@dhirsch1, I'm not sure what you mean if this applies to line 9. If line 6, understood.